### ISSUE TITLE
"You cannot call a method on a null-valued expression." on "Set up projects for analysis"

### ISSUE THREAD

**Issue by paulomorgado** [security-code-scan/security-code-scan-add-action#1]
I'm getting this error on "Set up projects for analysis"

```text
InvalidOperation: D:\a\_temp\238bcc4a-d7dd-49be-9a75-f51f8f0d60ab.ps1:94
Line |
  94 |    $propertyGroups[0].AppendChild($errorLog)
     |    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | You cannot call a method on a null-valued expression.

Error: Process completed with exit code 1.
```

Repo is this: https://github.com/paulomorgado/Microsoft.Extensions.AspNet

securitycodescan-analysis.yml file:
```yaml
# This workflow integrates SecurityCodeScan with GitHub's Code Scanning feature
# SecurityCodeScan is a vulnerability patterns detector for C# and VB.NET

name: SecurityCodeScan

on:
  push:
    branches: [ main ]
  pull_request:
    # The branches below must be a subset of the branches above
    branches: [ main ]
  schedule:
    - cron: '31 23 * * 6'

jobs:
  SCS:
    runs-on: windows-latest
    steps:
      - uses: actions/checkout@v2
      - uses: nuget/setup-nuget@v1.0.5
      - uses: microsoft/setup-msbuild@v1.0.2
      
      - name: Set up projects for analysis
        uses: security-code-scan/security-code-scan-add-action@main
        
      - name: Restore dependencies	
        run: dotnet restore

      - name: Build
        run: dotnet build --no-restore

      - name: Convert sarif for uploading to GitHub
        uses: security-code-scan/security-code-scan-results-action@main

      - name: Upload sarif
        uses: github/codeql-action/upload-sarif@v1
```

**Comment by paulomorgado** [comment]
Solved with https://github.com/paulomorgado/Microsoft.Extensions.AspNet/blob/b759ef0dc402fff2ead50f082f7fecb66c619f86/.github/workflows/securitycodescan-analysis.yml#L27-L140

I can submit a PR.

**Comment by JarLob** [comment]
Please create the PR. Thanks!

**Comment by JarLob** [comment]
There was the "always get the latest version" missing so I added it. There are some changes in behavior like "remove old SCS references", but I think it may be ok. Merged in d63f7dcd816ac2fbdf70e84c06a6761c1e79cb7f and 394f2853766c1fb78e60bca845efd8be8442df30. Thank you!
